.github/workflows/Create-NewReleases.yml

```diff
@@ -29,11 +29,11 @@ jobs:
         id: nextver
         run: |
           if [[ -f version.txt ]]; then
-            NEXT_TAG=$(cat version.txt)
+             NEXT_TAG=$(cat version.txt)
           else
             CURRENT_TAG=$(git tag --sort=-v:refname | head -n 1)
-            if [[ $TAG == '' ]]; then 
```

Comment:
Smart man, approved! Totally left this over from the old solution, I don't expect this to ever trigger (hopefully?) but still should be set correctly!